js/components/controlBar.js

```diff
@@ -553,6 +557,31 @@ var ControlBar = createReactClass({
         </div>
       ),
 
+      /**
+       * This function returns the chromecast button definition based on if
+       * it's connected to a chromecast or not
+       * @private
+       * @return {Object} The button definition
+       */
+      chromecast: function() {
+        let castIcon = "chromecast-disconnected";
+        if (this.props.controller.state.chromecast.isConnected === true)
+        {
+          castIcon = "chromecast-connected";
+        }
+        return (
+        <ControlButton
+          {...commonButtonProps}
+          key={CONSTANTS.CONTROL_BAR_KEYS.CHROMECAST}
+          focusId={CONSTANTS.CONTROL_BAR_KEYS.CHROMECAST}
+          ariaHidden={false}
```

Comment:
This only needed if set to `true`, I think it'd be better to remove it in this case